Content.Client/Commands/GuiShaderDevCommand.cs

```diff
@@ -0,0 +1,117 @@
+﻿#if DEBUG
+#nullable enable
+using System;
+using System.IO;
+using System.Linq;
+using Content.Client.Development;
+using JetBrains.Annotations;
+using Robust.Client.Graphics.Shaders;
+using Robust.Client.Interfaces.Console;
+using Robust.Client.Interfaces.UserInterface;
+using Robust.Shared.IoC;
+using Robust.Shared.Maths;
+using Robust.Shared.Prototypes;
+using YamlDotNet.RepresentationModel;
+
+namespace Content.Client.Commands
+{
+
+    [UsedImplicitly]
+    internal class GuiShaderDevCommand : IConsoleCommand
+    {
+
+        public string Command => "shaderDevWindow";
+
+        public string Help => "Usage: shaderDevWindow <name>\n" + Description;
+
+        public string Description => "Loads a shader onto a colored rectangle in a UI window.";
+
+        public bool Execute(IDebugConsole console, params string[] args)
+        {
+            var uiMan = IoCManager.Resolve<IUserInterfaceManager>();
+            var prototypeManager = IoCManager.Resolve<IPrototypeManager>();
+
+            switch (args.Length)
+            {
+                case 0:
+                {
+                    console.AddLine("Missing arguments.");
+                    return false;
+                }
+                default:
+                {
+                    console.AddLine("Invalid arguments.");
+                    return false;
+                }
+                case 1:
+                {
+                    if (!prototypeManager.HasIndex<ShaderPrototype>(args[0])) {
+                        console.AddLine("Shader '" + args[0] + "' cannot be found in shader prototypes!");
+                        return false;
+                    }
+                    var wnd = new SimpleShaderDevWindow(args[0]);
+                    wnd.OpenCentered();
+                    break;
+                }
```

Comment:
I get Shader prototype has no source exceptions when I try using this even when I checked out the engine PR. Probably need to handle those?